engine/lib/views.php

```diff
@@ -1690,6 +1690,26 @@ function elgg_views_boot() {
 
 	elgg_extend_view('elgg.css', 'colorbox.css');
 
+	// TODO 3.0 move the images
+	elgg_register_simplecache_view('colorbox-images/border1.png');
+	elgg_register_simplecache_view('colorbox-images/border2.png');
+	elgg_register_simplecache_view('colorbox-images/loading.gif');
+
+	// @deprecated 2.2 old colorbox CSS/image locations
+	// TODO can we do this somewhere else, not every request?
```

Comment:
We could do it when we flush caches. We could do an exception for these 3 views and write them to cache.